### ISSUE TITLE
Different input form behavior when trying to invoke firenvim via c-e with  takeover = 'never'

### ISSUE THREAD

**Issue by mjlbach** [glacambre/firenvim#868]
- OS Version: mac x86-64
- Browser Version:  Firefox 84.0
- Browser Addon Version: 0.2.2
- Neovim Plugin Version: 0.2.2
- What I tried to do: 

I prefer invoking firenvim manually to having automatic capture. With the standard configuration, firenvim captures GitHub's input form on issues correctly:

<img width="1280" alt="Screen Shot 2021-01-11 at 11 43 11 AM" src="https://user-images.githubusercontent.com/13316262/104230475-7e443880-5402-11eb-9909-18660184ba9d.png">

- What happened:
When I add the following to my init.lua:
```lua
vim.g.firenvim_config = { localSettings = { ['.*'] = { takeover = 'never' } } }
```
and trigger firenvim taking over the input form with C-e, it used to replicate the above behavior. After 0.21/0.22, instead the following occurs.

https://user-images.githubusercontent.com/13316262/104230635-acc21380-5402-11eb-89f8-1b2b5494c572.mov

Unrelated, but taking over the input form *only* occurs if I hover over the input without clicking.

**Comment by mjlbach** [comment]
Thank you!

**Comment by glacambre** [comment]
Hi, thanks for the report. This was a dumb mistake (I thought a DOM attribute was a boolean when it was actually a ternary). I'll publish a new release soon :)

**Comment by mjlbach** [comment]
Thank you for making something I use often enough to notice :laughing:

**Comment by glacambre** [comment]
I published v0.2.3 on the firefox add-on store, let me know if you still have issues :)

**Comment by mjlbach** [comment]
Works perfectly!

*sent from firenvim*
